game-core/src/test/java/games/strategy/triplea/TripleATest.java

```diff
@@ -0,0 +1,59 @@
+package games.strategy.triplea;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.collection.IsCollectionWithSize.hasSize;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import games.strategy.engine.chat.Chat;
+import games.strategy.engine.data.GameData;
+import games.strategy.engine.framework.ServerGame;
+import games.strategy.engine.framework.startup.launcher.LaunchAction;
+import games.strategy.engine.framework.startup.ui.PlayerType;
+import games.strategy.engine.player.Player;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+public class TripleATest {
+
+  private static final String DELEGATE_NAME_EDIT = "edit";
```

Comment:
Is this a constant in the production code? WDYT to use a constant that lives in prod code such that we do not couple the test?

Reply:
Actually I haven't found the "edit" String declared as a constant anywhere else and their using it directly `startGame`, which is kind of ugly too.